pkg/ebpf/c/tracee.bpf.c

```diff
@@ -2770,20 +2783,28 @@ static __always_inline struct pipe_inode_info *get_file_pipe_info(struct file *f
 SEC("raw_tracepoint/sys_enter")
 int tracepoint__raw_syscalls__sys_enter(struct bpf_raw_tracepoint_args *ctx)
 {
-    event_data_t data = {};
-    if (!init_event_data(&data, ctx))
+    struct task_struct *task = (struct task_struct *) bpf_get_current_task();
+    u64 task_id = bpf_get_current_pid_tgid(); // get the tgid only
+    task_info_t *task_info = NULL;
+
+    int zero = 0;
+    config_entry_t *config = (config_entry_t *) bpf_map_lookup_elem(&config_map, &zero);
```

Comment:
This lookup might not be necessary

Reply:
needed for submit checks, see [here](https://github.com/aquasecurity/tracee/pull/2080#discussion_r958348112)